### DISCUSSION TITLE
Pining is not working as expected

### DISCUSSION THREAD

**Discussion by softdrawart** [shteeve3d/blender-wiggle-2#74] [Q&A]
I need to animate a chain like structure, I have a chain of connected bones and one last bone has a (damped track) to another target bone/empty that it is pining to. When I play the animation with wiggle enabled and all bones wiggle (Bone Tail) enabled, last bone is stretching to the target pin bone/empty but all other bones in the chain do not rearange/move so that the last bone is not so stretched.
![image](https://github.com/user-attachments/assets/cd995afd-8498-41f1-9815-e7acd06841a1)
What I expect to look like is this:
![image](https://github.com/user-attachments/assets/1cf498ec-86a6-4102-ae2b-07cba5f26c4a)

**Comment by softdrawart** [comment]
I found that rasing the quality of the wiggle simulation fixes it) also adding extra limit scale constraint on the pined bone helps remove the stretchiness

**Reply by schroef** [reply]
Shouldnt you flip the direction of the bones.

**Reply by schroef** [reply]
PS you r solution works nice. I still see a bit weirdness. But my setup is a bit awkward i guess

https://github.com/user-attachments/assets/c39ede60-61f5-491c-b29d-65670ad1ab10
